lms/djangoapps/shoppingcart/utils.py

```diff
@@ -69,7 +69,7 @@ def parse_lt_objects(lt_objects):
     for lt_object in lt_objects:
         if isinstance(lt_object, LTTextBox) or isinstance(lt_object, LTTextLine):
             # text
-            text_content.append(lt_object.get_text().encode('utf-8'))
```

Comment:
Is `lt_object` already a byte string?  This call to encode was meant to make sure that the thing we are adding to text_content is bytes of encoded unicode.

Reply:
```
in python2.7 
(Pdb) lt_object.get_text()
u'Order # 1\nDate: Aug 22, 2019\n'
(Pdb) type(lt_object.get_text())
<type 'unicode'>
(Pdb) type(lt_object.get_text().encode('utf-8'))
<type 'str'>
```

in python3
```
(Pdb) lt_objects[0].get_text()
'Invoice\n'
(Pdb) lt_objects[0].get_text().encode('utf-8')
b'Invoice\n
```

We don't need bytes here otherwise join will break.
```        
 return '\n'.join(text_content)
E       TypeError: sequence item 0: expected str instance, bytes found
```